src/types/redis_zset.cc

```diff
@@ -931,14 +931,6 @@ rocksdb::Status ZSet::RandMember(const Slice &user_key, int64_t command_count,
 }
 
 rocksdb::Status ZSet::Diff(const std::vector<Slice> &keys, MemberScores *members) {
-  std::vector<std::string> lock_keys;
```

Comment:
Hi @git-hulk , I have omitted the locking in the ZSet::Diff. is this what you had in mind?

Reply:
Yes, I'm good with this change. We can improve this in the future by using the same snapshot between keys in RangeByScore.